### PR TITLE
Database to terraform 12

### PR DESCRIPTION
**Pull request by The-smooth-operator**
Upgrading this to Terraform 0.12 to start using it for Voice

### CODE REVIEW

**Comment by mozafrank** [comment]
Ah yeah I ran into the same version keyword issue. :+1:

File: aws/database/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# AWS Database module
+Use this module for deploying RDS instances in AWS
+
+## Usage
+Example on how to use module
+
+```
+module "database" {
+	# TODO: Fill me
```
Comment (mozafrank) [inline aws/database/README.md]: @The-smooth-operator Can you replace this TODO before merging?

Comment (The-smooth-operator) [reply aws/database/README.md]: oh sure I forgot to add it. Thanks for catching this

**Comment by The-smooth-operator** [comment]
@mozafrank thanks for the catch. I've added the example now

**Comment by The-smooth-operator** [comment]
yes. I also run it again and Terraform made no changes to formatting

**Pull request merged**